domain/src/main/java/me/androidbox/domain/authentication/model/Event.kt

```diff
@@ -0,0 +1,14 @@
+package me.androidbox.domain.authentication.model
+
+data class Event(
+    val id: String,
+    val title: String,
+    val description: String,
+    val startDateTime: Long,
+    val endDateTime: Long,
+    val remindAt: Long,
+    val host: String, /* event creator ID */
```

Comment:
If you feel like you need a comment to describe a field in a domain model, why not directly rename it to `eventCreatorId`?

Reply:
@philipplackner Updated to `eventCreatorId`